src/mango/test/mango.py

```diff
@@ -220,7 +226,7 @@ def bulk_delete(self, docs):
 
     def find(self, selector, limit=25, skip=0, sort=None, fields=None,
                 r=1, conflicts=False, use_index=None, explain=False,
-                bookmark=None, return_raw=False, update=True, executionStats=False):
+                bookmark=None, return_raw=False, update=True, executionStats=False, stale=None):
```

Comment:
instead of `stale=Ok`. What about using `update`? That is what we are testing here. https://github.com/apache/couchdb/blob/master/src/mango/src/mango_cursor_view.erl#L393

Reply:
stale=ok looks to be quite used by user instead of `update`. want to leave it for now

Reply:
Since `update=False` is equal to `stale=ok` and already exists, let me use `update=False`.